zone/gm_commands/npcspawn.cpp

```diff
@@ -20,33 +31,24 @@ void command_npcspawn(Client *c, const Seperator *sep)
 	bool is_delete = !strcasecmp(sep->arg[1], "delete");
 	bool is_remove = !strcasecmp(sep->arg[1], "remove");
 	bool is_update = !strcasecmp(sep->arg[1], "update");
+	bool is_clone = !strcasecmp(sep->arg[1], "clone");
 	if (
 		!is_add &&
 		!is_create &&
 		!is_delete &&
 		!is_remove &&
-		!is_update
+		!is_update &&
+		!is_clone
 	) {
-		c->Message(Chat::White, "Command Syntax: #npcspawn [Add|Create|Delete|Remove|Update]");
+		c->Message(Chat::White, "Command Syntax: #npcspawn [Add|Create|Delete|Remove|Update|Clone|Help] [optional 3rd parameter]");
```

Comment:
No need to specify the third parameter here, you can make note of it within the help messages.

Reply:
Resolved and tested in new [commit](https://github.com/EQEmu/Server/pull/4311/commits/6524e60ba154786a0f7a5727e0717644fbd6a1d3)